src/Bridge/CommonMark/CommonMarkParser.php

```diff
@@ -3,23 +3,23 @@
 namespace Mni\FrontYAML\Bridge\CommonMark;
 
 use League\CommonMark\CommonMarkConverter;
-use League\CommonMark\MarkdownConverterInterface;
+use League\CommonMark\ConverterInterface;
 use Mni\FrontYAML\Markdown\MarkdownParser;
 
 /**
  * Bridge to the League CommonMark parser
  */
 class CommonMarkParser implements MarkdownParser
 {
-    private MarkdownConverterInterface $parser;
+    private ConverterInterface|null $parser;
 
-    public function __construct(MarkdownConverterInterface $commonMarkConverter = null)
+    public function __construct(ConverterInterface|null $commonMarkConverter = null)
```

Comment:
Just to be sure: this isn't a BC break?

Reply:
Hmm, good question. I'd not thought of that. Will check.

Reply:
I'll admit that I cannot say with 100% certainty that it isn't. However, I believe it's not as, up until now, the parameter has been allowed to be null, marking it as a null type _implicitly_. I believe that this change just makes that state explicit.

Reply:
No I mean the change from `MarkdownConverterInterface` to `ConverterInterface`

Reply:
Sorry, I misunderstood. Thanks for clarifying. Will look into it today.

Reply:
I hope that I've better understood you this time. ConverterInterface has been available since [2.2.0](https://github.com/thephpleague/commonmark/releases/tag/2.2.0), back in early 2022. I suspect that it likely wouldn't be a BC break, but hope that I've assessed this correctly.

Reply:
This is a BC break, we can't merge that as-is.

Here's an example:

```php
// This is userland code

$parser = new ...(); // class that implements MarkdownConverterInterface

new Mni\FrontYAML\Bridge\CommonMark\CommonMarkParser($parser);
```

the code above works today. With your PR, it wouldn't work.